src/main/java/ReviewInteractors/DeleteReviewRequestModel.java

```diff
@@ -0,0 +1,46 @@
+//Class responsible for bundling input data for delete use case
+
+package ReviewInteractors;
+
+import ReviewInterfaces.ReviewGatewayInterface;
+import ReviewInterfaces.UserGatewayInterface;
+import restaurant_use_case.RestaurantDSGateway;
+import entities.Review;
+import entities.User;
+import entities.Restaurant;
+
+public class DeleteReviewRequestModel {
+    private final ReviewGatewayInterface reviewGateway;
+    private final UserGatewayInterface userGateway;
+    private final RestaurantDSGateway restaurantGateway;
+    private final Review review;
+    private final User user;
+    private final Restaurant restaurant;
+
```

Comment:
We should discuss how our request models should look for consistency, because I pass through the gateways and presenters when constructing the interactor before passing into controller, and the request model just contains the information needed to change the Object.